src/index.html

```diff
@@ -22,6 +22,405 @@
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo header"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+              data-qa="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            class="link"
+            data-qa="hover"
+          >
+            <a
+              href="index.html"
+              class="nav-item"
+              data-qa="nav-hover"
+            >
+              LAPTOPS &AMP; COMPUTERS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card-container">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="card"
+        />
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__info">Product code: 195434</p>
+        <div class="reviews">
+          <div class="stars stars--0">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="reviews__text">Reviews: 5</div>
+        </div>
+        <div class="price">
+          <div class="price__title">Price:</div>
+          <div class="price__number">$2,199</div>
+        </div>
+        <footer
+          class="footer"
+          data-qa="card-hover"
+        >
+          <a
+            class="footer__button"
+            href="#buy"
```

Comment:
Consider adding a more descriptive href value for the 'BUY' button to improve navigation. Currently, it links to '#buy', which may not be meaningful or functional.